drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -144,6 +148,22 @@ def self.static_options_for_type(pick_list_type, user:)
       end
     end
 
+    def self.eligible_staff_assignment_user_picklist(current_user)
+      return [] unless current_user
+
+      # Check that the user has can_edit_enrollments permission on at least one project with staff_assignment_enabled
```

Comment:
to discuss: Is this permission check correct / is it overkill?

Reply:
This pick list type should accept project as an argument, and the returned list should only include users with access to edit this enrollment.

This means we can get rid of the with_staff_assignments_enabled scope